README.md

```diff
@@ -3,106 +3,214 @@
-* No Virtual DOM – you can use any other library with Catberry without conflicts
+## What the cat is that?
+
+Catberry was developed to help in creating ["isomorphic/Universal" Web applications](https://github.com/catberry/catberry/blob/8.0.0-dev/docs/index.md#isomorphicuniversal-applications).
+
+Long story short, isomorphic/universal applications are apps that use shared codebase in both server and browser environments and can render pages working as [Single Page Application](http://en.wikipedia.org/wiki/Single_Page_Application) in a browser and as a back-end application on the server using the same modules.
+
+## TLDR;
+
+Install [Catberry CLI](https://www.npmjs.com/package/catberry-cli) using following command:
+
+```bash
+npm install -g catberry-cli
+```
+
+Now you can create an empty project with [Handlebars](http://handlebarsjs.com/) support like this:
+
+```bash
+catberry init empty-handlebars
+```
+
+Or an example application that works using GitHub API:
+
+```bash
+catberry init example
+```
+
+Also, you can get a list of all templates:
+
+```bash
+catberry init ?
+```
+
+## Useful links
+
+* [Catberry Documentation](https://github.com/catberry/catberry/blob/8.0.0-dev/docs/index.md)
+* [Get Started Guide](https://github.com/catberry/catberry/blob/8.0.0-dev/docs/index.md#get-started)
+* [Officially supported plugins](https://github.com/catberry/catberry/blob/8.0.0-dev/docs/index.md#https://github.com/catberry/catberry/blob/%23221-es6-docs/docs/index.md#list-of-officially-supported-plugins)
+* [Catberry's homepage](http://catberry.org) and its [source code](https://github.com/catberry/catberry-homepage)
+* [Todo application](https://github.com/catberry/catberry-todomvc)
+* [Example application](https://github.com/catberry/catberry-cli/tree/master/templates/example)
+
+## Why should I use that?
+
+### Architecture
+
+* The entire architecture of the framework is built using the [Service Locator](https://github.com/catberry/catberry/blob/8.0.0-dev/docs/index.md#service-locator) pattern that helps managing module dependencies and [create plugins](https://github.com/catberry/catberry/). Also,   [Flux](https://github.com/catberry/catberry/blob/8.0.0-dev/docs/index.md#flux) is used as a data layer
+* [Cat-components](https://github.com/catberry/catberry/blob/8.0.0-dev/docs/index.md#cat-components) – similar to [web-components](http://webcomponents.org/) but organized as directories, can be rendered on the server and published/installed as NPM packages
+* Catberry builds a bundle for running the application in a browser as a [Single Page Application](http://en.wikipedia.org/wiki/Single_Page_Application)
+* ES2015/ES6 support using [Babel](http://babeljs.io/)
+* The whole framework's API uses [Promises](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Promise)
+* Framework itself is an [express](https://github.com/visionmedia/express)/[connect](https://github.com/senchalabs/connect)
```

Comment:
* Framework itself is an [express](https://github.com/visionmedia/express)/[connect](https://github.com/senchalabs/connect) middleware, which means you can use it with other [middlewares](http://expressjs.com/en/guide/using-middleware.html)